README.md

```diff
@@ -81,12 +83,16 @@ Make sure `node` is on your `PATH`. If it's not, some options:
 
 ##### Fedora and other RPM-based distributions
 
+See the [generic Linux notes above](https://github.com/abandonware/noble/blob/master/README.md#linux) first.
+
 ```sh
 sudo yum install bluez bluez-libs bluez-libs-devel
 ```
 
 ##### Intel Edison
 
+See the [generic Linux notes above](https://github.com/abandonware/noble/blob/master/README.md#linux) first.
```

Comment:
may we use relative links instead ?

Reply:
This will work i think:
[generic Linux notes above](README.md#linux)

Reply:
This works